lib/ChipApi.php

```diff
@@ -124,6 +124,17 @@ public function chargePurchase($purchaseId, $token) {
 			]
 		]), new Model\Purchase());
 	}
+
+	/**
+	 * 
+	 * @param \Chip\Model\ClientDetails $client
+	 * @return \Chip\Model\ClientDetails
+	 */
+	public function createClient($client) {
+		return $this->mapper->map($this->request('POST', 'client/', [
```

Comment:
the route requires to be in plural.

change it from:

```php
return $this->mapper->map($this->request('POST', 'client/', [
```

to:

```php
return $this->mapper->map($this->request('POST', 'clients/', [
```

Reply:
Thanks for notice @wzul . Have updated it